python/katana/cpp/libgalois/graphs/Graph.pxd

```diff
@@ -154,6 +155,7 @@ cdef extern from "katana/Graph.h" namespace "katana" nogil:
 
         shared_ptr[CTable] node_properties()
         shared_ptr[CTable] edge_properties()
+        EntityTypeManager& GetNodeTypeManager()
```

Comment:
Should you also provide `GetEdgeTypeManager()`?

Reply:
I don't really care about edge types right now for what I'm doing, but yes, it's probably a good idea.